### ISSUE TITLE
Use _.runInContext

### ISSUE THREAD

**Issue by jdalton** [atomantic/undermore#11]
For the bit here
```js
 _ = require('lodash');
module.exports = mixins;
```

you can do:
```js
 _ = require('lodash').runInContext();
module.exports = mixins;
```

this will create a pristine copy of lodash so you can augment it without affecting the actual `lodash` package.

**Comment by atomantic** [comment]
Ooh, great catch! Thanks for pointing that out. I'll update my docs :)

**Comment by atomantic** [comment]
fixed in 1.9.1: https://github.com/atomantic/undermore/commit/f469942de1e0301c5e8c075033e58765503ea1a8
